packages/examples/packages/get-file/src/index.ts

```diff
@@ -21,12 +22,11 @@ import { bytesToString, hexToBytes } from '@metamask/utils';
 export const onRpcRequest: OnRpcRequestHandler = async ({ request }) => {
   switch (request.method) {
     case 'getFile': {
-      const fileInHexadecimal = await snap.request({
+      const fileInPlaintext = await snap.request({
         method: 'snap_getFile',
-        params: { path: './src/foo.json', encoding: 'hex' },
+        params: { path: './src/foo.json', encoding: 'utf8' },
```

Comment:
I wonder if we should make it default to UTF-8. That's what `Buffer.from` does at least. 🤔

Reply:
Base64 will be the most performant since it is also the storage format. I think it makes sense to default to that